uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/microsoftsts/MicrosoftStsPromptHandler.java

```diff
@@ -93,6 +93,12 @@ public void handlePrompt(@NonNull final String username, @NonNull final String p
         final IMicrosoftStsLoginComponentHandler aadLoginComponentHandler =
                 (IMicrosoftStsLoginComponentHandler) loginComponentHandler;
 
+         if (parameters instanceof MicrosoftStsPromptHandlerParameters &&
+                ((MicrosoftStsPromptHandlerParameters) parameters).isUseSignInWithADifferentAccount() &&
+                !((MicrosoftStsPromptHandlerParameters) parameters).isFederated()) {
+             ((AadLoginComponentHandler) aadLoginComponentHandler).handleSignInWithDifferentAccount(username, password);
```

Comment:
You probably need to check that it is indeed instance of `AadLoginComponentHandler` because it can be anything that extends `IMicrosoftStsLoginComponentHandler`